pages/downloads.md

```diff
@@ -2,7 +2,7 @@
 title: Source Downloads
 weight: 4
 ---
-{% assign baseUrl = 'https://www.apache.org/dyn/closer.cgi?path=incubator/zipkin' %}
```

Comment:
we are required to use a mirror according to @nacx. Also the links work fine

Reply:
ok it seems for some reason a chinese mirror is not propagating the same files. 

ex. https://www-eu.apache.org/dist/incubator/zipkin/zipkin-api/0.2.1/apache-zipkin-api-incubating-0.2.1-source-release.zip.sha512
fine, but
https://mirrors.tuna.tsinghua.edu.cn/apache/incubator/zipkin/zipkin-api/0.2.1/apache-zipkin-api-incubating-0.2.1-source-release.zip.sha512
not fine

the fix would be to do two URLs, mirror for download and non-mirror for asc/512 with a comment (if supported in this syntax) to say that some mirrors don't propagate all files